### ISSUE TITLE
Allow for importing images

### ISSUE THREAD

**Issue by julienw** [julienw/forms-extension#3]
See the branch https://github.com/julienw/forms-extension/tree/import-image

**Comment by Natim** [comment]
You mean as a signature?

**Comment by julienw** [comment]
yep; currently we can draw directly on the screen but not import signatures.
I started something but never finished
